ohm-js/test/test-ohm-syntax.js

```diff
@@ -976,6 +976,47 @@ test('inheritance', t => {
   t.end();
 });
 
+test('override with "..."', t => {
+  let g = ohm.grammar('G { letter := "@" | ... }');
+  t.equal(g.match('@', 'letter').succeeded(), true);
+  t.equal(g.match('a', 'letter').succeeded(), true);
+
```

Comment:
... and another test that ensures that the "after terms" get evaluated after the "...". How about this?

```
let g = ohm.grammar('G { letter := ... | "ab" }');
t.equal(g.match('a', 'letter').succeeded(), true);
t.equal(g.match('ab', 'letter').succeeded(), false);
```

Reply:
Done.